### ISSUE TITLE
[FEAT] Conversion Functions with Default Values?

### ISSUE THREAD

**Issue by KiddoV** [gookit/goutil#111]
Would be nice to have some utils to convert to a design data type, if error or not the same type then return the default vals:

I have a scenarios where I want to convert a float and want the value to be ``0.0`` if failed:
```go
test := mathutil.QuietFloat(math.NaN) //=> test = NaN, I want it to always be a float type instead (0.0 if failed).
``` 

The idea...
```go
utils.FloatOrDefault(in any, default float64)
test := utils.FloatOrDefault("test", 0.0) //=> Error, return test = 0.0

utils.StringOrDefault(in any, default string)
test := utils.StringOrDefault(nil, "hello") //=> Error, return test = "hello"

...
```

Something like that. you get the idea 🙂 
Thanks,

**Comment by inhere** [comment]
Hi @KiddoV  added and released on [v0.6.12](https://github.com/gookit/goutil/releases/tag/v0.6.12)

**Comment by KiddoV** [comment]
Nice. Thank you!
